### PR TITLE
updated docs for v0.23.10 release

### PR DESCRIPTION
**Pull request by Huang-Wei**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

- update README.md to include the latest image URLs
- now users are encouraged to use `multiPoint` in v1beta3 (old v1beta2 config are still supported)
- update the install doc
- simplified Helm charts by switching to v1beta3 config. template values and
  docs are also updated

#### Which issue(s) this PR fixes:

Part of #406.

#### Special notes for your reviewer:

Followup:

- [ ] Cherry-pick this to release-1.23.

### CODE REVIEW

File: manifests/install/charts/as-a-second-scheduler/templates/deployment.yaml [hunk 1]
```diff
@@ -42,20 +42,20 @@ spec:
...
         - /bin/kube-scheduler
-        - --address=0.0.0.0
-        - --leader-elect=false
         - --config=/etc/kubernetes/scheduler-config.yaml
         image: {{ .Values.scheduler.image }}
         livenessProbe:
           httpGet:
             path: /healthz
-            port: 10251
+            port: 10259
```
Comment (Huang-Wei) [inline manifests/install/charts/as-a-second-scheduler/templates/deployment.yaml:50]: Kube scheduler doesn't server http traffic starting 1.23: https://github.com/kubernetes/website/pull/31419

File: manifests/install/charts/as-a-second-scheduler/values.yaml [hunk 2]
```diff
@@ -5,31 +5,20 @@
...
 # as they need extra RBAC privileges on metrics.k8s.io.
 
 plugins:
-  enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]
-
-global:  
-   # queueSort is not indented under extensions 
-   # as it needs to be globally enabled once. 
-  queueSort: ["Coscheduling"]
-  extensions:
```
Comment (Huang-Wei) [inline manifests/install/charts/as-a-second-scheduler/values.yaml:28]: @helayoty FYI: with the latest v1beta3 config, we can simplify the Helm parameters by just specifying which plugins should be enabled/disabled.

File: doc/install.md [hunk 3]
```diff
@@ -88,26 +88,11 @@ any vanilla Kubernetes scheduling capability. Instead, a lot of extra out-of-box
     profiles:
     - schedulerName: default-scheduler
       plugins:
-        queueSort:
+        multiPoint:
```
Comment (Huang-Wei) [inline doc/install.md:91]: FYI: @damemi config using multiPoint is pretty neat.

File: manifests/install/charts/as-a-second-scheduler/values.yaml (hunk 2 above)
Comment (helayoty) [reply manifests/install/charts/as-a-second-scheduler/values.yaml:28]: much better :)

File: manifests/install/charts/as-a-second-scheduler/templates/configmap.yaml [hunk 4]
```diff
@@ -6,43 +6,23 @@ metadata:
...
-        {{- end }}
-      {{- end }}
-
- {{- end }}
+          {{- range $.Values.plugins.disabled }}
+          - name: {{ title . }}
+          {{- end }}
 
   {{- /* TODO: wire CRD installation with enabled plugins. */}}
+{{- end }}
```
Comment (yibozhuang) [inline manifests/install/charts/as-a-second-scheduler/templates/configmap.yaml:28]: nit: newline

File: manifests/install/charts/as-a-second-scheduler/values.yaml [hunk 5]
```diff
@@ -5,31 +5,20 @@
...
-  extensions: 
-    preFilter: ["Coscheduling", "CapacityScheduling"] 
-    filter: ["NodeResourceTopologyMatch"] 
-    postFilter: ["Coscheduling", "CapacityScheduling"] 
-    score: ["NodeResourceTopologyMatch", "NodeResourcesAllocatable"] 
-    permit: ["Coscheduling"] 
-    reserve: ["Coscheduling", "CapacityScheduling"] 
-    postBind: ["Coscheduling"]
+  enabled: ["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch","NodeResourcesAllocatable"]
+  disabled: ["PrioritySort"] # only in-tree plugins need to be defined here
```
Comment (yibozhuang) [inline manifests/install/charts/as-a-second-scheduler/values.yaml:24]: nit: newline

File: manifests/install/charts/as-a-second-scheduler/README.md [hunk 6]
```diff
@@ -32,23 +31,16 @@ scheduler-plugins-scheduler    1/1     1            1           7s
...
-| `global.extensions.postBind`            | `postBind` extension config. This is be used if the `postBind` plugin is enabled.                                                                                                                | `["Coscheduling"]`                                      |
+| Parameter                               | Description                   | Default                                                                                         |
+| --------------------------------------- |-------------------------------|-------------------------------------------------------------------------------------------------|
+| `scheduler.name`                        | Scheduler name                | `scheduler-plugins-scheduler`                                                                   |
+| `scheduler.image`                       | Scheduler image               | `k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.23.10`                                          |
```
Comment (yibozhuang) [inline manifests/install/charts/as-a-second-scheduler/README.md:37]: nit: can we align the Default column?

**Comment by Huang-Wei** [comment]
/label tide/merge-method-squash

@yibozhuang updated.

**Comment by Huang-Wei** [comment]
@denkensk mind LGTMing this?

**Comment by denkensk** [comment]
/lgtm

Thanks @Huang-Wei

**Pull request merged**